packages/design/tailwind/css/components/button.css

```diff
@@ -118,3 +118,13 @@
 .gi-btn-regular {
   @apply gi-h-10 gi-px-3 gi-py-2 gi-text-sm;
 }
+
+/* Button Group */
+
+.gi-btn-group {
+  @apply gi-flex gi-flex-wrap gi-gap-3 gi-justify-center sm:gi-justify-start;
```

Comment:
Stretch the items so they will all have the same width.
Consider using a grid since we have to control the behaviour of items when shown on mobile.